datafusion/physical-expr/src/aggregate/first_last.rs

```diff
@@ -44,6 +44,7 @@ pub struct FirstValue {
     expr: Arc<dyn PhysicalExpr>,
     ordering_req: LexOrdering,
     requirement_satisfied: bool,
+    is_ignore_null: bool,
```

Comment:
maybe we can use `ignore_nulls: bool`, it is consistent with SQL keywords and also window functions that in parallel implementing IGNORE NULLS .

Would be also nice to have a comments in struct about this field

Reply:
Fixed. Thanks